pkg/grpc/user_devices.proto

```diff
@@ -26,6 +26,7 @@ service UserDeviceService {
       returns (RegisterUserDeviceFromVINResponse);
   rpc UpdateDeviceIntegrationStatus(UpdateDeviceIntegrationStatusRequest)
       returns (UserDevice);
+  rpc IssueVinCredential(VinCredentialRequest) returns (VinCredentialResponse);
```

Comment:
I think we should draw some distinction in the request/response here that this is for the issuing endpoint. Like `VinCredentialResponse` could be confused with `VinCredential`. I got them confused!